tests/postgres_tests/test_schemas.py

```diff
@@ -0,0 +1,23 @@
+from django.test import modify_settings
+
+from . import PostgreSQLTestCase
+from .models import SchemaModel
+
+
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class SchemaTest(PostgreSQLTestCase):
+
+    Model = SchemaModel
+
+    def setUp(self):
+        self.Model.objects.bulk_create([
+            self.Model(int1=1),
+            self.Model(int1=2),
+            self.Model(int1=3),
+        ])
+
+    def test_values(self):
+        index = 1
+        for object in self.Model.objects.all():
+            self.assertEqual(object.int1, index)
+            index += 1
```

Comment:
I believe there were some cases where you need the table name alone, but I really can't recall which those were (maybe it was about moving a table from one schema to another...). Now that I think of this, it's likely select/update/insert/delete queries won't need special handling for schema qualified tables.

By the way, right now it's already possible to use schema qualified tables with unofficial syntax:
class Foo(models.Model):
    ...
    class Meta:
        db_table = '"foo"."bar"'

What I did for the schema qualified tables branch was using schema qualified tables everywhere in the tests. This is doable by hacking model's meta properties dynamically after model initialisation. This gives a nice list of cases that are do not work. Some of those cases can't work, but most are legitimate issues. Fixing all of those isn't a requirement, but to me trying to find out what doesn't yet work is a requirement.